application-templates/django-ninja/README.md

```diff
@@ -0,0 +1,86 @@
+# __APP_NAME__
+
+Django-Ninja/React-based web application.
+This application is constructed to be deployed inside a cloud-harness Kubernetes.
+It can be also run locally for development and test purpose.
+
+The code is generated with the script `harness-application`.
+
+## Configuration
+
+### Accounts
+
+The CloudHarness Django application template comes with a configuration that can retrieve user account updates from Keycloak (accounts)
+To enable this feature:
+* log in into the accounts admin interface
+* select in the left sidebar Events
+* select the `Config` tab
+* enable "metacell-admin-event-listener" under the `Events Config` - `Event Listeners`
+
+An other option is to enable the "metacell-admin-event-listener" through customizing the Keycloak realm.json from the CloudHarness repository.
+
+## Develop
+
+This application is composed of a Django-Ninja backend and a React frontend.
+
+### Backend
+
+Backend code is inside the *backend* directory.
+See [backend/README.md#Develop]
+
+### Frontend
+
+Backend code is inside the *frontend* directory.
```

Comment:
DOC: `Frontend code`?

Reply:
Yep, should be, one moment...

Reply:
Fixed, same issue was present in the existing django template, so that's been updated as well.

Reply:
thanks @condar-metacell !